app/views/admin/sites/_form.html.haml

```diff
@@ -18,6 +18,12 @@
   = f.text_field :ad_slot
   = f.text_area :head_tag
   = f.text_area :promotion_tag
+  = f.form_group label: {text: Site.human_attribute_name(:theme_color)} do
```

Comment:
Add help text using `help` option, please.
If you cannot describe in English, please describe in Japanese on Slack or esa.io.